spread.yaml

```diff
@@ -829,35 +829,41 @@ suites:
             # Enable secure boot in the nested vm in case it is supported
             ENABLE_SECURE_BOOT: '$(HOST: echo "${SPREAD_ENABLE_SECURE_BOOT:-false}")'
             # Build and use snapd from current branch
-            BUILD_SNAPD_FROM_CURRENT: '$(HOST: echo "${SPREAD_BUILD_SNAPD_FROM_CURRENT:-false}")'
+            BUILD_SNAPD_FROM_CURRENT: '$(HOST: echo "${SPREAD_BUILD_SNAPD_FROM_CURRENT:-true}")'
             # Download and use an custom image from this url
             CUSTOM_IMAGE_URL: '$(HOST: echo "${SPREAD_CUSTOM_IMAGE_URL:-}")'
             # Directory where the images and test assets are stored
             WORK_DIR: '/tmp/work-dir'
         manual: true
         kill-timeout: 40m
         debug: |
-            if [ -f "${WORK_DIR}/serial.log" ]; then
-                cat "${WORK_DIR}/serial.log"
+            #shellcheck source=tests/lib/nested.sh
+            . "$TESTSLIB/nested.sh"
```

Comment:
Sourcing this sets 

```
WORK_DIR="${WORK_DIR:-/tmp/work-dir}"
NESTED_VM=nested-vm
SSH_PORT=8022
MON_PORT=8888
```
None of those are prefixed with anything. This is very easy to get confusing. Could we prefix all of those with NESTED_ at minimum?

Reply:
Update that, now all the global vars are prefixed with NESTED_
Thanks for the comment